### DISCUSSION TITLE
Amiga

### DISCUSSION THREAD

**Discussion by flagone** [OnionUI/Onion#537] [Q&A]
I got Amiga working once on an older release of the Onion OS but cannot get it working again.

Is it correct that the Kickstart Roms are to be named exactly as in the Wiki and placed in the BIOS folder?

That is, should they be called "kick33180.A500, kick34005.A500 and kick40068.A1200"?  

When I try to load a game I just get the image of the upside disk and the game doesn't start up.

If I try the other Amiga emulator available in Expert mode, it complains I don't have the Kickstart Roms and crashes out to a cursor.

**Comment by Olywa** [comment]
Bit late to your question but maybe it helps. I think the WIKI only covers the very basic/essential kickstarts. 

- I have it working by using the full list of kickstarts covered here: https://docs.libretro.com/library/puae/
- The MD5 value is also there and you can use an application like "Md5Checker" to ensure you have the correct files
- Your files should be named correctly as per the link and placed in the root of the /BIOS/ folder & also in a subfolder /BIOS/Kickstarts/

Before this I was also getting the upside-down disk complaint and now everything I've tried runs.

**Reply by flagone** [reply]
I've got it loading games again.  Not entirely sure why.  I messed around with Bios files for a while trying everything I had but no luck.  Then I went to the bottom of the Bios files and found some Whdload files which I deleted.  I also deleted a Puae file in the Roms directory.  Not sure where they came from but it is working again.

I very much appreciate your assistance.

**Comment by Olywa** [comment]
Awesome, glad you were able to get it sorted.
